example-app/src/main/res/values-night/colors.xml

```diff
@@ -10,7 +10,7 @@
     <color name="color_divider">#1AFFFFFF</color>
 
     <color name="text_color">@color/color_white</color>
-
+    <color name="textColor">@color/color_white</color>
```

Comment:
This does not really fix the problem, but hides it. `@color/textColor` should not be referenced directly by styles and views. Instead they should use `?android:attr/textColor`

Reply:
Correct me if I'm wrong but these styes are in the example app, the SDK styles still do not reference `@color/textColor` 🤔

Reply:
`AdyenCheckout.SimpleTextItemTextView` and `AdyenCheckout.AddressForm.HeaderTextAppearance` are referencing `@color/textColor` and shouldn't